src/events.ts

```diff
@@ -68,13 +68,14 @@ export const handleDeletedEvent = async event => {
   return db.queryAsync(query, [event.id, event]);
 };
 
-export async function sendEvent(event, to) {
+export async function sendEvent(event, to, method = 'POST') {
   event.token = sha256(`${to}${serviceEventsSalt}`);
   event.secret = sha256(`${to}${serviceEventsSalt}`);
   const headerSecret = sha256(`${to}${process.env.SERVICE_EVENTS_SALT}`);
+  const url = to.replace('[PROPOSAL-ID]', event.id.split('/')[1]);
```

Comment:
I feel this URL format is only useful for our case, others may not use it or we need to find a way to send event type

Reply:
True. We can and should add support to inject all the other parameters send in the post body as a query params placeholder. And we should also support head request